### PR TITLE
OSOE-70: Adapting to Helpful Libraries refactoring

### PR DESCRIPTION
**Pull request by Piedone**
OSOE-70

### CODE REVIEW

File: Lombiq.ChartJs.csproj [hunk 1]
```diff
@@ -39,16 +39,18 @@
 
   <ItemGroup>
     <PackageReference Include="OrchardCore.Module.Targets" Version="1.3.0" />
+    <PackageReference Include="OrchardCore.DisplayManagement" Version="1.3.0" />
+    <PackageReference Include="OrchardCore.ResourceManagement" Version="1.3.0" />
```
Comment (sarahelsaig) [inline Lombiq.ChartJs.csproj]: Why?

Comment (Piedone) [reply Lombiq.ChartJs.csproj]: To make sure tag helpers work.

Comment (sarahelsaig) [reply Lombiq.ChartJs.csproj]: Yeah, but iirc we agreed to put that into the OrchardCore specific HelpfulLibraries project so we don't have to include these packages in each module.

Comment (Piedone) [reply Lombiq.ChartJs.csproj]: Yep, and I did that, but this project only uses a couple of features from the `Common`, not `OrchardCore` HL project.

Comment (sarahelsaig) [reply Lombiq.ChartJs.csproj]: I don't see the advantage, becuase this is an OC module.

Comment (Piedone) [reply Lombiq.ChartJs.csproj]: Fair enough.

**Review by sarahelsaig — APPROVED** [review]

**Pull request merged**